linker/src/main/resources/start-template.sh

```diff
@@ -44,13 +44,14 @@ usage() {
 main() {
     local action command
     init "$@"
+    # shellcheck disable=SC2086
     ${action} ${command}
 }
 
 init() {
     local -r scriptName=$(basename "${0}")
     local -r binDir=$(dirname "${0}")
-    local -r homeDir=$(cd "${binDir}/.."; pwd)
+    local -r homeDir=$(cd "${binDir}/.." || exit; pwd)
```

Comment:
This file is technically shipped to end-users, can you triple check that these changes won't break ?